package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nm-stylus-library",
-  "version": "0.9.0",
+  "version": "0.10.0",
```

Comment:
If you are going to run `npm version` to iterate this then you actually want to revert this back to 0.9.0 before merging. Hopefully as I'm making this PR review just a comment not a request for changes it will say it can be merged by you after you change this...